src/Bicep.Core.Samples/Files/baselines/Modules_CRLF/main.sourcemap.bicep

```diff
@@ -2377,13 +2377,13 @@ module withSeparateConfig './child/folder with separate config/moduleWithAzImpor
 //@            "_generator": {
 //@              "name": "bicep",
 //@              "version": "dev",
-//@              "templateHash": "8139539847401806108"
+//@              "templateHash": "2846749952698758252"
 //@            }
 //@          },
 //@          "imports": {
 //@            "az": {
 //@              "provider": "AzureResourceManager",
-//@              "version": "1.0.0"
+//@              "version": "builtin"

```

Comment:
Was this change intentional? There's no corresponding change to `src/Bicep.Core.Samples/Files/baselines/Modules_CRLF/main.bicep`.

Reply:
Yeah, its intentional, the value of the version is not defined in `main.bicep`, currently the test is setup to load the builtin provider since the thing being exercised is not the provider loading capability.

Once we get the nuget package and the provider package version to be the same, then we can have the version reflected to be the builtin provider nuget version. and  that is what we should specify when we exercise a subject under test that is not testing the loading functionality